### ISSUE TITLE
When examining the interaction, the covalent bond is not displayed.

### ISSUE THREAD

**Issue by kaishi-08** [gnina/gnina#254]
When I pulled up the results on Discovery Studio to observe the interaction, instead of forming a covalent bond, it displayed as an unfavorable bump. I utilized a command targeting a specific Residue, namely Cys, and a suitable warhead group.

**Comment by dkoes** [comment]
Please provide a reproducible example and a better description of what the problem is.

**Comment by kaishi-08** [comment]
Thank sir. Here is one of several results I obtained when targeting ATOM S at Cys81. However, upon examining the interaction between the receptor and its ligand (through Discovery Studio), there is no formation of a covalent bond. This is the issue I am currently facing. Thank you for taking the time to assist me.

          (_)            
   __ _ _ __  _ _ __   __ _ 
  / _` | '_ \| | '_ \ / _` |
 | (_| | | | | | | | | (_| |
  \__, |_| |_|_|_| |_|\__,_|
   __/ |                    
  |___/                     

gnina v1.1 master:e4cb380+   Built Dec 18 2023.
gnina is based on smina and AutoDock Vina.
Please cite appropriately.

Commandline: ./gnina -r 2lnsA.pdb -l NOHA-1.pdb --config ConfigCys.txt --covalent_rec_atom A:81:SG --covalent_lig_atom_pattern C=C -o Output_ligand/NOAH-1a_lig_covalent_docking.sdf
Using random seed: 144604580

0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
***************************************************

mode |  affinity  |  intramol  |    CNN     |   CNN
     | (kcal/mol) | (kcal/mol) | pose score | affinity
-----+------------+------------+------------+----------
    1       -2.83        0.00       0.5767      2.740
    2       -2.77        0.00       0.5631      3.206
    3       -3.14        0.00       0.5284      3.037
    4       -2.54        0.00       0.5270      2.939
    5       -2.73        0.00       0.5242      3.274
    6       -2.78        0.00       0.5203      2.928
    7       -3.29        0.00       0.5174      3.486
    8       -2.96        0.00       0.4820      2.960
    9       -3.94        0.00       0.4797      3.284

Below is the image illustrating the interaction between the ligand and the receptor.

![image](https://github.com/gnina/gnina/assets/166632424/3a8b7c47-2b61-4e55-aa0c-a6777f044856)

Thank you.

**Comment by dkoes** [comment]
I can't reproduce your example since you haven't provided any files, but of course there is going to be a steric clash at the covalent interaction if the ligand is being analyzed as a non-covalent binder.

**Comment by kaishi-08** [comment]
> Please provide a reproducible example and a better description of what the problem is.






> I can't reproduce your example since you haven't provided any files, but of course there is going to be a steric clash at the covalent interaction if the ligand is being analyzed as a non-covalent binder.

Thank sir. I used the warhead group to form a covalent bond. These are the files I used in the example above.
[Covalent Docking.zip](https://github.com/gnina/gnina/files/15140431/Covalent.Docking.zip)

**Comment by dkoes** [comment]
There is nothing wrong here.  The output is correct.  Why do you think otherwise?

**Comment by kaishi-08** [comment]
> There is nothing wrong here. The output is correct. Why do you think otherwise?

Thank you. I've only recently started delving into this field, so there are still many mistakes on my part. I hope you understand. Regarding this issue, when I observed the interaction between the ligand (output file) and the receptor (2lnsA), I didn't see any covalent bonds formed in the image I posted above; it indicated an unfavorable bump instead. Therefore, I think I might have made a mistake somewhere along the way. Thank you for taking the time to assist me.
